babel/src/test/resources/sql/big-query.iq

```diff
@@ -2462,6 +2462,14 @@ FROM Dates;
 
 !ok
 
+# Show that the return type matches the operand type
+SELECT
+    DATE_TRUNC(TIMESTAMP "2008-12-25 15:30:00", MONTH) AS timestamp_result,
+    DATE_TRUNC(DATETIME "2008-12-25 15:30:00", MONTH) AS datetime_result;
+timestamp_result TIMESTAMP_WITH_LOCAL_TIME_ZONE NOT NULL
+datetime_result TIMESTAMP NOT NULL
+!type
```

Comment:
neat, I didnt know about `!type` in quidem tests

Reply:
yeah super useful!!